qelem/lib/screens/question_detail_screen.dart

```diff
@@ -0,0 +1,177 @@
+import 'package:flutter/material.dart';
+import 'package:qelem/widgets/answer_card.dart';
+
+import '../widgets/answer_bottom_sheet.dart';
+
+class QuestionDetailScreen extends StatefulWidget {
+  const QuestionDetailScreen({Key? key}) : super(key: key);
+
+  @override
+  State<QuestionDetailScreen> createState() => _QuestionDetailScreenState();
+}
+
+class _QuestionDetailScreenState extends State<QuestionDetailScreen> {
+  var dropdownValue = "Recency";
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: NestedScrollView(
+        body: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  child: Padding(
+                    padding: const EdgeInsets.symmetric(horizontal: 8.0),
+                    child: ListView(
+                      children: const [
+                        AnswerCard(),
+                        AnswerCard(),
+                        AnswerCard(),
+                        AnswerCard(),
+                        AnswerCard(),
+                        AnswerCard(),
+                        AnswerCard(),
+                        AnswerCard(),
+                        AnswerCard(),
+                      ],
+                    ),
+                  ),
+                )
+              ],
+            ),
+            Positioned(
+                right: 0.0,
+                bottom: 0.0,
+                child: GestureDetector(
+                  onTap: () {
+                    showModalBottomSheet(
+                      context: context,
+                      isScrollControlled: true,
+                      builder: (builder) {
+                        return const AnswerBottomSheet();
+                      },
+                    );
+                  },
+                  child: Container(
+                    decoration: BoxDecoration(
+                      color: Theme.of(context).primaryColor,
+                      borderRadius: const BorderRadius.only(
+                        topLeft: Radius.circular(20.0),
+                      ),
+                    ),
+                    width: 98,
+                    height: 44,
+                    child: Center(
+                      child: Text("ANSWER",
+                          style: Theme.of(context).primaryTextTheme.button),
+                    ),
+                  ),
+                ))
+          ],
+        ),
+        headerSliverBuilder: (BuildContext context, bool innerBoxIsScrolled) {
+          return <Widget>[
+            SliverAppBar(
```

Comment:
Why did you make the app bar collapsable?

Reply:
To utilize the screen space better when reading answers.